tests/bench_e2e_fp8.py

```
@@ -0,0 +1,315 @@
+import time
+
+import torch
+
+# Ensure pyhie is importable from repo
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'python'))
+sys.stdout.reconfigure(line_buffering=True)
+
+from dashinfer import allspark
+from dashinfer.allspark.engine import TargetDevice
+from dashinfer.allspark.prompt_utils import PromptTemplate
+from dashinfer.allspark._allspark import GenerateRequestStatus, AsCacheMode
```

Flake8 (7.3.0)

[error] 'dashinfer.allspark._allspark.AsCacheMode' imported but unused

(F401)